source/includes/_release-notes.html.md

```diff
@@ -0,0 +1,77 @@
+# Product Release Notes
+
+These release notes combine releases for the web (BRAIN Dashboard), the backend services (Bonsai AI Engine), and Bonsai's API. All releases are tagged by date for the web, backend, and API. Releases are based on version for the SDKs and CLI which are available on GitHub.
```

Comment:
This line "Releases are based on version for the SDKs and CLI which are available on GitHub." This had to be explained to me. What was confusing is that I read 'Releases' as being Platform releases, but it's referring to SDK and CLI versions. I would suggest instead: "SDK and CLI are released separately and have different version numbers. They are available on GitHub."